plugins/view-transitions/includes/settings.php

```diff
@@ -284,14 +317,20 @@ static function (): void {
 			'description' => __( 'Enable view transitions in the WordPress admin area.', 'view-transitions' ),
 		),
 	);
+
+	// Do not render the checkbox to override if there is no thing to override.
```

Comment:
```suggestion
	// Do not render the checkbox to override if there is nothing to override.
```

Reply:
Fixed in https://github.com/WordPress/performance/pull/2037/commits/2431d7a68d6dc1149dd84389bb555cd1bdab7422